ssh-keygen.c

```diff
@@ -1541,12 +1541,18 @@ do_change_comment(struct passwd *pw)
 	sshkey_free(private);
 
 	strlcat(identity_file, ".pub", sizeof(identity_file));
-	fd = open(identity_file, O_WRONLY | O_CREAT | O_TRUNC, 0644);
```

Comment:
Understand this is a constructive change. But we prefer not to make any changes to base code to reduce merge conflicts. We are not anyway near to integrate this upstream.

Reply:
Ok, got it.